code/game/objects/items/ego_weapons/he.dm

```diff
@@ -1429,3 +1432,124 @@
 
 /obj/item/ego_weapon/lance/split/get_clamped_volume()
 	return 40
+
+/obj/item/ego_weapon/lance/lifestew_lance
+	name = "lifetime stew"
+	desc = "Salutations! I implore thee to try this soup!"
+	icon_state = "lifestew" //placeholder
+	lefthand_file = 'icons/mob/inhands/96x96_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/96x96_righthand.dmi'
+	inhand_x_dimension = 96
+	inhand_y_dimension = 96
+	force = 35
+	reach = 2		//Has 2 Square Reach.
+	attack_speed = 2.0 // really slow
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	attack_verb_continuous = list("burns", "boils")
+	attack_verb_simple = list("burn", "boil")
+	hitsound = 'sound/weapons/fixer/generic/fire1.ogg'
+	attribute_requirements = list(
+							TEMPERANCE_ATTRIBUTE = 40
+							)
+	charge_speed_cap = 3 //Charges slower, weaker overall while charging, dealing less damage at a full charge. But it has an AOE!
+	force_per_tile = 2
+	pierce_force_cost = 15
+
+/obj/item/ego_weapon/lance/lifestew_lance/UserBump(mob/living/carbon/human/user, atom/A)
+	. = ..()
+	if(charge_speed <= -(charge_speed_cap / 2)) //At a decent charge level, it'll do this once.
+		charge_speed += (2 * pierce_speed_cost)
+		if(isliving(A))
+			SoupBlast(user, A)
+
+/obj/item/ego_weapon/lance/lifestew_lance/proc/SoupBlast(mob/living/carbon/human/user, mob/target)
+	playsound(target, 'sound/abnormalities/bloodbath/Bloodbath_EyeOn.ogg', 80, TRUE, -3) //yes im reusing a sound bite me
+	for(var/turf/T in view(1, target))
+		var/obj/effect/temp_visual/small_smoke/halfsecond/FX =  new(T)
+		FX.color = "#622F22"
+		for(var/mob/living/L in T)
+			if(L == user || ishuman(L))
+				continue
+			user.HurtInTurf(T, list(), 40, BLACK_DAMAGE, check_faction = TRUE)
```

Comment:
```suggestion
		user.HurtInTurf(T, list(), 40, BLACK_DAMAGE, check_faction = TRUE)
```
Previous version would hurt everyone in the turf for each being in the turf. Not good.